romancal/resample/resample.py

```diff
@@ -191,101 +195,113 @@ def do_drizzle(self):
         else:
             return self.resample_many_to_one()
 
-    def resample_many_to_many(self):
-        """Resample many inputs to many outputs where outputs have a common frame.
+    def resample_group(self, input_models, indices):
+        """Apply resample_many_to_many for one group
 
-        Coadd only different detectors of the same exposure (e.g. map SCA 1 and
-        10 onto the same output image), as they image different areas of the
-        sky.
+        Parameters
+        ----------
+        input_models : ModelLibrary
 
-        Used for outlier detection
+        indices : list
```

Comment:
Consider adding a one-line description of what the indices correspond to (I realize I didn't do any of these docstring updates for the jwst code either...)

Reply:
How does b8ba8af look?

Reply:
looks good except (nitpick) the second instance of "and will not" could be changed to "nor"

Reply:
I'm going to leave this as-is in part because I can't think of a good way to fit in a "nor". I know it's considered a universal gate so it should be able to do anything (bad joke)!

Reply:
sounds good